gridstatus/decorators.py

```diff
@@ -387,3 +390,12 @@ def __ladd__(self, other):
 
     def __radd__(self, other):
         return self.__ladd__(other)
+
+class FiveMinOffset:
+    def __ladd__(self, other):
+        # round up to next 5 min interval
+        # remove seconds and microseconds
+        return other.replace(second=0, microsecond=0) + pd.DateOffset(minutes=5) - pd.DateOffset(minutes=other.minute % 5)
```

Comment:
You can round up a datetime/Timestamp to the nearest 5 min with `.ceil("5min")`

Reply:
Cool I'll do that in the next iteration just stepped away from my computer